src/preferences/listener.ts

```diff
@@ -19,16 +24,56 @@ export function createConfigChangeListener(): Disposable {
       const configs: WorkspaceConfiguration = workspace.getConfiguration();
 
       if (event.affectsConfiguration(SSL_PEM_PATHS)) {
+        // inform the sidecar that the SSL/TLS .pem paths have changed
         logger.debug(`"${SSL_PEM_PATHS}" config changed`);
         const pemPaths: string[] = configs.get(SSL_PEM_PATHS, []);
         await updatePreferences({
           tls_pem_paths: pemPaths,
         });
-      } else if (event.affectsConfiguration(SSL_VERIFY_SERVER_CERT_DISABLED)) {
+        return;
+      }
+
+      if (event.affectsConfiguration(SSL_VERIFY_SERVER_CERT_DISABLED)) {
```

Comment:
There could be a case where both `event.affectsConfiguration(SSL_PEM_PATHS)` is true and `event.affectsConfiguration(SSL_VERIFY_SERVER_CERT_DISABLED)` is true, correct? just checking since this moves from `else if` to `if`

Reply:
We would only see one change event come in at a time (one for each setting adjustment)